modules/ROOT/pages/automated-policy-apply.adoc

```diff
@@ -5,6 +5,20 @@ endif::[]
 
 Users with *Organization Admin*, *Environment Admin*, or with a role to *Manage Policies* in the environment can create, edit and delete Automated Policies.
 
+== Task Prerequisites
+
+Before you apply an automated policy, ensure that:
+
+* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
+* An xref:api-instance-landing-page.adoc[API], configured with a basic endpoint or a proxy endpoint, exists in API Manager.
+* A Mule application is deployed with a xref:connectors::http/http-listener-ref.adoc[HTTP(s) based flow], which is linked to the API through Autodiscovery, also known as tracked flow.
```

Comment:
Does "tracked flow" refer to the Autodiscovery process or the deployment as a whole? This is not clear. If it refers to the deployment as a whole, I suggest "This deployment process is also called a tracked flow.

Reply:
A flow is said to be "tracked" if it has an associated autodiscovery element linking it to a valid API.

"When [autodiscovery](https://docs.mulesoft.com/api-manager/2.x/api-auto-discovery-new-concept) is correctly configured in your Mule application you can say that your application’s API is tracked by or paired to API Manager."

Reply:
Done! Thanks!